File: csharp/get_calendarview.cs
var client = new RestClient("https://graph.microsoft.com/v1.0/me/calendar/calendarView?endDateTime=2017-01-07T19%3A00%3A00.0000000&startDateTime=2017-01-01T19%3A00%3A00.0000000");
var request = new RestRequest(Method.GET);
IRestResponse response = client.Execute(request);

File: java/get_calendarview.java
HttpResponse<String> response = Unirest.get("https://graph.microsoft.com/v1.0/me/calendarView?endDateTime=2016-10-01T19%3A00%3A00.0000000&startDateTime=2016-01-01T19%3A00%3A00.0000000")
  .asString();
